src/cli.rs

```diff
@@ -354,20 +356,6 @@ pub(crate) async fn poll_for_user_input<E: EventHandler>(
 				}
 				"nodeinfo" => node_info(&channel_manager, &peer_manager),
 				"listpeers" => list_peers(peer_manager.clone()),
-				"signmessage" => {
```

Comment:
We should be able to add this back in if we add a `NodeSigner` method to sign a message.